src/components/beers.js

```diff
@@ -0,0 +1,48 @@
+import React,{useState, useEffect} from 'react';
+import axios from 'axios';
+import '../components/allbears.css'
```

Comment:
Since both the component and the stylesheet are in the same folder, you could just write
```
import './allbears.css';